packages/react-composites/src/composites/common/Drawer/MoreDrawer.tsx

```diff
@@ -252,6 +257,30 @@ export const MoreDrawer = (props: MoreDrawerProps): JSX.Element => {
     });
   }
 
+  /* @conditional-compile-remove(raise-hand) */
+  if (drawerSelectionOptions !== false && isEnabled(drawerSelectionOptions?.raiseHandButton)) {
+    const raiseHandIconName = raiseHandButtonProps.checked
+      ? 'LowerHandContextualMenuItem'
+      : 'RaiseHandContextualMenuItem';
+    drawerMenuItems.push({
+      itemKey: 'raiseHandButtonKey',
+      disabled: props.disableButtonsForHoldScreen || isDisabled(drawerSelectionOptions.raiseHandButton),
+      text: raiseHandButtonProps.checked
+        ? localeStrings.component.strings.raiseHandButton.onLabel
+        : localeStrings.component.strings.raiseHandButton.offLabel,
+      onItemClick: () => {
+        if (raiseHandButtonProps.onToggleRaiseHand) {
+          raiseHandButtonProps.onToggleRaiseHand();
+        }
+        onLightDismiss();
+      },
+      iconProps: {
+        iconName: raiseHandIconName,
```

Comment:
nit: better name `raiseHandButtonIcon` we dont want to include the suffix `name` in the icons

Reply:
changed